tiledb/sm/consolidator/consolidator.cc

```diff
@@ -155,8 +155,9 @@ void Consolidator::array_consolidate(
   }
 
   if (array_uri.is_tiledb()) {
-    throw_if_not_ok(storage_manager->rest_client()->post_consolidation_to_rest(
-        array_uri, config));
+    throw_if_not_ok(
+        storage_manager->resources().rest_client()->post_consolidation_to_rest(
```

Comment:
`class Consolidator` has a member variable `resources_`.

Reply:
This is a static function - I created a followup story to remove the `StorageManager*` arguments from the `Consolidator` code in favor of `ContextResources&`

Reply:
OK